mkdocs.yml

```
@@ -39,3 +47,4 @@ nav:
   - Use of data elements: data_elements.md
   # defer to gen-files + literate-nav
   - Code Reference: reference/
+  - Package workflows: package_workflows.md
```

This is rendering strangely, moving it above the Code Reference fixed it.

```suggestion
  - Package workflows: package_workflows.md
  # defer to gen-files + literate-nav
  - Code Reference: reference/
```